db/data/regions/BE.yml

```diff
@@ -31,6 +31,9 @@ format:
   show: "{firstName} {lastName}_{company}_{address1}_{address2}_{zip} {city}_{country}_{phone}"
 format_extended:
   edit: "{country}_{firstName}{lastName}_{company}_{streetName}{streetNumber}_{address2}_{zip}{city}_{phone}"
+address1_regex:
+  - "^(?<streetName>[^\\d,]+),? (?<streetNumber>\\d+(?: ?[A-Za-z])?)$"
+  - "^(?<streetNumber>\\d+(?: ?[A-Za-z])?),? (?<streetName>[^\\d,]+)$"
```

Comment:
The second pattern matches french addresses, which typically have the street number first